.yarn/plugins/local/plugin-lifecycle-scripts.js

```diff
@@ -0,0 +1,72 @@
+// @ts-check
+
+/**
+ * Run an array of tasks in sequence.
+ *
+ * @param tasks {Array<Function>} - An array of functions that return a function
```

Comment:
The code 
```suggestion
 * @param tasks {Array<(() => Promise<number>)>} - An array of functions that return a function
```

The documentation says that those are functions that return functions returning a promise, but the code is only a function returning promises, without the second function return, which is correct?

Reply:
Good catch, I changed this later but didn't update the documentation. Done now.

`Array<() => Promise<number>` doesn't seem to be valid in jsdoc.